gpkit/build.py

```diff
@@ -279,7 +271,12 @@ def build(self):
 
 def build_gpkit():
     "Builds GPkit"
-    global settings  # pylint: disable=global-variable-not-assigned
+    try:
+        import gpkit
+        log("# Moving to the directory from which GPkit was imported.")
+        os.chdir(gpkit.__path__[0])
```

Comment:
hmm, does this have the potential to cause problems if there are multiple installs of GPkit and the user wants to run the build script to overwrite the currently-installed one?